src/ConnectionConfig.tsx

```diff
@@ -36,11 +36,19 @@ export const ConnectionConfig: FC<ConnectionConfigProps> = (props: ConnectionCon
   if (profile === undefined) {
     profile = options.database;
   }
+  const tempCredsFeatureEnabled = config.featureToggles.awsDatasourcesTempCredentials;
 
   const settings = (window as any).grafanaBootData.settings;
   const awsAllowedAuthProviders = useMemo(
-    () => settings.awsAllowedAuthProviders ?? [AwsAuthType.Default, AwsAuthType.Keys, AwsAuthType.Credentials],
-    [settings.awsAllowedAuthProviders]
+    () =>
+      settings.awsAllowedAuthProviders ?? [
+        AwsAuthType.Default,
+        AwsAuthType.Keys,
+        AwsAuthType.Credentials,
+        AwsAuthType.GrafanaAssumeRole,
```

Comment:
I don't think you want this here right?

Reply:
No, I must have forgotten to delete line 48 🙄 good catch!